jobs/frontend/senior-frontend-developer.md

```diff
@@ -0,0 +1,58 @@
+# Senior Frontend Developer
+
+We're looking for a pro-active senior-level frontend developer, who not only solves tasks but suggests improvements on the codebase level as well as on product and company level, and who shares knowledge within the team on meetups and in chats. <b> We are open to a person with ambitions to fill in the Team Lead role. </b>
+
+## About the job
+
+- Salary: up to 5000USD, gross per month.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually, our projects last for about 1.5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- For development we usually use Kanban or similar process in Trello, Pivotal or Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use Holacracy for organizational governance.
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## We expect that you
+
+- Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important
+- Can design a whole system of application
+- Have experience working with different kind of teams and projects
+- Research new technologies and promote them inside the team (new libraries, tools, techniques: refactoring, debugging, etc.)
+- Will constantly suggest innovations on the project
+- It will be a plus if you released own projects
+- Understand yourself: have good habits, know how to be productive and how to rest
+- Have knowledge of English on the level which allows you to feel comfortable communicating in it (verbally and written) the whole working day.
+- Know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication. It is important for us.
```

Comment:
```suggestion
## We expect that you

- Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important.
- Can design a whole system of application.
- Knows at least one library/framework for building UI at advanced level (React, Vue, Angular, etc.).
- Have experience working with different kind of teams and projects.
- Research new technologies and promote them inside the team (new libraries, tools, techniques: refactoring, debugging, etc.).
- Will constantly suggest innovations on the project.
- It will be a plus if you released own projects.
- Understand yourself: have good habits, know how to be productive and how to rest.
- Have knowledge of English on the level which allows you to feel comfortable communicating in it (verbally and written) the whole working day.
- Know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication. It is important for us.
```

I added dots at the end of each sentence because there were used mixed formats (some items have dot, some don't) and it's better to use one format. Besides that, I added `Knows at least one library/framework for building UI at advanced level (React, Vue, Angular, etc.).` because I think it's important for a candidate to know at least one library/framework at advanced level and makes it clearer that it doesn't matter what library/framework they use so it might increase amount of applicants.